lib/modules/platform/gerrit/utils.ts

```diff
@@ -89,14 +109,15 @@ export function mapGerritChangeStateToPrState(
     case 'ABANDONED':
       return 'closed';
   }
-  return 'all';
```

Comment:
are you sure this can never happen in future?

Reply:
Yes. The Gerrit REST API documentation is pretty clear, `status` is always set (because it is not marked as _optional_, and it can only have `NEW`, `MERGED`, or `ABANDONED`.

https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#change-info

If it ever happens in the future for some very unexpected reason, we better not mask it in my opinion.